### ISSUE TITLE
Buttons with display set to "NIL" lose their display value on reopen

### ISSUE THREAD

**Issue by qiemem** [NetLogo/NetLogo#672]
To reproduce:
1. Create a new model.
2. Create two buttons.
3. Set one button's display to "NIL", leave the other's blank.
4. Set code of both to `ca` or something.
5. Note that one does in fact read "NIL" and the other "ca"
6. Save. Reopen

Result: The buttons both say "ca"

In the nlogo, the buttons look like:

```
BUTTON
31
45
94
78
NIL
ca
NIL
1
T
OBSERVER
NIL
NIL
NIL
NIL
1
```

```
BUTTON
37
166
100
199
NIL
ca
NIL
1
T
OBSERVER
NIL
NIL
NIL
NIL
1
```

Can you tell which is which? Neither can NetLogo.

Not sure this is worth fixing, but figured it should be documented. Our internal representation of NILable values should probably be done with `Option` instead.

**Comment by Idloj** [comment]
And how would it be written in the model file?

**Comment by mrerrormessage** [comment]
This problem is inherent in the ".nlogo" file format as used over the past dozen or so years. This will only be fixed if and when https://github.com/NetLogo/NetLogo/issues/849 is.

**Comment by mrerrormessage** [comment]
Closing since this can't be fixed without a new file format (captured by #849)
